scripts/upload_test_resources.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import argparse
+import logging
+import sys
+import traceback
+from pathlib import Path
+
+from quilt3 import Package
+
+from aicsimageio import __version__
+
+###############################################################################
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="[%(levelname)4s: %(module)s:%(lineno)4s %(asctime)s] %(message)s",
+)
+log = logging.getLogger(__name__)
+
+###############################################################################
+# Args
+
+
+class Args(argparse.Namespace):
+    def __init__(self):
+        self.__parse()
+
+    def __parse(self):
+        # Setup parser
+        p = argparse.ArgumentParser(
+            prog="upload_test_resources",
+            description=(
+                "Upload files used for testing this project. This will upload "
+                "whatever files are currently found in the `tests/resources` directory."
+                "To add more test files, simply add them to the `tests/resources` "
+                "directory and rerun this script."
+            ),
+        )
+
+        # Arguments
+        p.add_argument(
+            "--dry-run",
+            action="store_true",
+            help=(
+                "Conduct dry run of the package generation. Will create a JSON "
+                "manifest file of that package instead of uploading."
+            ),
+        )
+        p.add_argument(
+            "--debug",
+            action="store_true",
+            help="Show traceback if the script were to fail.",
+        )
+
+        # Parse
+        p.parse_args(namespace=self)
+
+
+###############################################################################
+# Build package
+
+
+def upload_test_resources(args: Args):
+    # Try running the download pipeline
+    try:
+        # Get test resources dir
+        resources_dir = (
+            Path(__file__).parent.parent / "aicsimageio" / "tests" / "resources"
```

Comment:
```suggestion
            Path(__file__).parent / "tests" / "resources"
```

Reply:
Nope. This file lives in scripts so it needs to go up two levels from current file. First parent is scripts itself then second parent is aicsimageio repo directory, then down one to aicsimageio python package directory. upside_down_face

Reply:
(see other comment)